mods/default/nodes.lua

```diff
@@ -586,30 +586,121 @@ minetest.register_node("default:snow", {
 	inventory_image = "default_snowball.png",
 	wield_image = "default_snowball.png",
 	paramtype = "light",
+	paramtype2 = "leveled",
 	buildable_to = true,
 	floodable = true,
+	walkable = true,
 	drawtype = "nodebox",
+	leveled = 8,
```

Comment:
16

Reply:
Done!

Reply:
Doesn't seem done?

Reply:
@An0n3m0us Did you forget to push?

Reply:
Oops, sorry I'd forgot to commit and push it xD